test cases/d/9 features/meson.build

```
@@ -1,4 +1,4 @@
-project('D Features', 'd', default_options : ['debug=false'])
+project('D Features', 'd', meson_version: '>=1.5.99', default_options : ['debug=false'])
```

```suggestion
project('D Features', 'd', meson_version: '>=1.6', default_options : ['debug=false'])
```
The project test runner has some magic code to handle `.99` versions gracefully. (using either `./run_single_test.py` or `./run_project_tests.py`)

---

Ah cool, I didn't know that. Fix applied in the latest version.